db/migrate/20160830110050_change_cloud_subnets_network_ports_unique_index.rb

```diff
@@ -0,0 +1,15 @@
+class ChangeCloudSubnetsNetworkPortsUniqueIndex < ActiveRecord::Migration[5.0]
+  def change
+    remove_index :cloud_subnets_network_ports,
+                 :column => [:cloud_subnet_id, :network_port_id],
+                 :name   => 'index_cloud_subnets_network_ports',
+                 :unique => true
+
+    add_index :cloud_subnets_network_ports, :address
+
+    add_index :cloud_subnets_network_ports,
+              [:cloud_subnet_id, :network_port_id, :address],
+              :name   => 'index_cloud_subnets_network_ports_address',
+              :unique => true
```

Comment:
@Fryguy Actually I am adding a unique indexes to all N:M relation tables I am creating. I like my DB integrity :-) But yeah, the downside is that we can't backport changes. But usually, there is no usecase for having the same M:N relation multiple times in the relation table.

The Darga backport was solved by limiting the data in AWS refresh.